### ISSUE TITLE
NVDA testing workbench reads many times clickable when tabbing through editor tabs

### ISSUE THREAD

**Issue by dbaeumer** [Microsoft/vscode#16982]
VSCode Version: 1.8.0-insider

- activate nvda
- click through open editor tabs

You here the word clickable at least 3 times when tabbing onto an editor tab

**Comment by bpasero** [comment]
Looks like a regression from the Chrome update to me. I cannot reproduce with Stable.

**Comment by bpasero** [comment]
Still reproduces all the way to Chrome 66 (Electron 3.0.x).

**Comment by bpasero** [comment]
Duplicate of https://github.com/Microsoft/vscode/issues/60120
